Scenario Based Samples/Iframe Full Server Embedding/README.md

```diff
@@ -0,0 +1,88 @@
+ 
+    ![Default Authentication](https://github.com/boldbi/samples/assets/129487075/a81894ac-c147-41df-8c97-ed9928d16953)
+    
+ * Click on `Configuration` in UMS Settings, then select the `config.xml` file from the dropdown.  Change the value of the `<EnableSameTabLinkTarget>` tag from `false` to `true` in order to render the URL within the same application and `Save` the changes.
+    
+    ![Configuration](https://github.com/boldbi/samples/assets/129487075/b3d65c3c-d470-4f53-9983-a35291bb6bfe)
+
+ * Open the Administration Site Settings page in Bold BI Server. Under `Authentication`, select the `General` tab and enable `Enable Default Authentication`, then `Save` the changes. (ex: http://localhost:50000/bi/site/site1/administration)
+    
+    ![Admin Authentication](https://github.com/boldbi/samples/assets/129487075/763ae9a6-4448-40c3-9897-afb28db745ed)
+
+ * In the application, change the `jwt:boldbiserverurl` value in the `appsettings.json` file to the URL of our Bold BI server.
+  
+    ![appsettings.json](https://github.com/boldbi/samples/assets/129487075/efb08c80-ffcc-453e-b216-0949b06b9126)
+
+    
+ ## Run a Sample Using Command Prompt 
+    
+  1. Open the command prompt and navigate to the specified file [location](https://github.com/boldbi/samples/tree/master/Scenario%20Based%20Samples/Iframe%20Full%20Server%20Embedding/IframeFullServer) where the project is located.
+
+  2. Execute the command `dotnet restore` to restore the necessary packages. Once the packages have been successfully restored, use the `dotnet build` command to build the project.
+  
+  3. Finally, run the application using the command `dotnet run`.  It will display a URL typically something like (e.g., http://localhost:40000).  Copy this URL and paste it into your default web browser.
+
+ ## Developer IDE
+
+  * [Visual studio code](https://code.visualstudio.com/download)
+  
+### Run a Sample Using Visual Studio Code
+ 
+  * Open the folder `IframeFullServer` in Visual Studio Code.
+  
+  * Open the terminal in Visual Studio Code and execute the command `dotnet restore` to restore the required dependencies.
+ 
+  * Build your .NET project by executing the `dotnet build` command in the terminal.
+ 
+  * To run the application, use the command `dotnet run` in the terminal. After executing the command, the application has started and it will display a URL typically something like (e.g., http://localhost:40000).  Copy this URL and paste it into your default web browser.
+    
+    ![Bold BI Server image](https://github.com/boldbi/samples/assets/129487075/37e0af6a-bd96-478c-ae91-5948840f257d)
+
+    > **NOTE:** To mitigate issues related to NuGet packages, run the following command in terminal `dotnet add package Microsoft.IdentityModel.Tokens` and `dotnet add package System.IdentityModel.Tokens.Jwt`
```

Comment:
Note should not parallel to image. Must be like below.
![image](https://github.com/boldbi/samples/assets/92368346/66016a7c-7cb9-4271-adb9-8442637bd2f0)

Reply:
updated